assets/src/styles/blocks/Columns.scss

```diff
@@ -1,6 +1,8 @@
 .columns-block {
   a {
-    font-weight: inherit;
+    &:not(.btn) {
```

Comment:
Since we use `a` for some buttons in this block, it's hard to avoid using `:not`. We can iterate on that when we fix the markup.